libc/src/__support/FPUtil/generic/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ add_header_library(
     sqrt.h
     sqrt_80_bit_long_double.h
   DEPENDS
-    libc.include.fenv
+    libc.hdr.fenv_macros
```

Comment:
Check if the above 2 header files do not include `fenv_macros.h` then this dependency is not needed.